packages/test/test-end-to-end-tests/src/test/gc/gcUnreferencePhases.spec.ts

```diff
@@ -0,0 +1,133 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+import { requestFluidObject } from "@fluidframework/runtime-utils";
+import {
+    createSummarizerWithContainer,
+    ITestContainerConfig,
+    ITestObjectProvider,
+    mockConfigProvider,
+    summarizeNow,
+    waitForContainerConnection,
+} from "@fluidframework/test-utils";
+import { describeNoCompat, ITestDataObject, TestDataObjectType } from "@fluidframework/test-version-utils";
+import { IGCRuntimeOptions } from "@fluidframework/container-runtime";
+import { delay, stringToBuffer } from "@fluidframework/common-utils";
+import { gcTreeKey } from "@fluidframework/runtime-definitions";
+import { SummaryType } from "@fluidframework/protocol-definitions";
+import { getGCStateFromSummary, getGCTombstoneStateFromSummary } from "./gcTestSummaryUtils";
+
+/**
+ * Validates that an unreferenced datastore and blob goes through all the GC phases without overlapping.
+ */
+describeNoCompat("GC unreference phases", (getTestObjectProvider) => {
+    const inactiveTimeoutMs = 100;
+    const sweepTimeoutMs = 200;
+
+    const settings = {};
+    const gcOptions: IGCRuntimeOptions = {
+        inactiveTimeoutMs,
+        sweepTimeoutMs,
```

Comment:
This option doesn't do anything. GC doesn't sweepTimeoutMs as a runtime option.

Reply:
Fixed